README.md

```diff
@@ -104,22 +103,28 @@ All commands below will assume that your dummy project name is `local`
 
 Create the following topics/subscriptions:
 
-      Topic                          | Subscription
```

Comment:
This README is getting too long.  You don't have to fix this now, but it would be good to bite the bullet sometime and split out the details . See https://github.com/ONSdigital/sdc-int-mock-service/tree/main/ README for an example. 
The "firestore contention" section and the "manual testing" sections are obvious contenders for their own MD files.   I prefer to put these in a separate `doc` directory.

Reply:
I've extracted the 3 mentioned sections into their own MD files